Sources/SessionManager/SessionManager.swift

```diff
@@ -10,32 +10,29 @@ import curveSecp256k1
 
 public class SessionManager {
     private var sessionServerBaseUrl = "https://session.web3auth.io/v2/"
-    private var sessionID: String? {
-        didSet {
-            if let sessionID = sessionID {
-                KeychainManager.shared.save(key: .sessionID, val: sessionID)
-            }
-        }
-    }
+    private var sessionId: String?
 
     private let sessionNamespace: String = ""
     private let sessionTime: Int
     private let allowedOrigin: String
 
-    public func getSessionID() -> String? {
-        return sessionID
+    public func getSessionId() -> String? {
+        return sessionId
     }
 
-    public func setSessionID(_ val: String) {
-        sessionID = val
+    public func saveSessionId(_ sessionId: String) {
+        if !sessionId.isEmpty {
+            self.sessionId = sessionId
+            KeychainManager.shared.save(key: .sessionID, val: sessionId)
+        }
     }
 
     public init(sessionServerBaseUrl: String? = nil, sessionTime: Int = 86400, sessionID: String? = nil, allowedOrigin: String? = "*") {
-        if let sessionID = sessionID {
-            self.sessionID = sessionID
+        if let sessionId = sessionId {
+            self.sessionId = sessionId
         } else {
-            if let sessionID = KeychainManager.shared.get(key: .sessionID) {
-                self.sessionID = sessionID
+            if let sessionId = KeychainManager.shared.get(key: .sessionID) {
+                self.sessionId = sessionId
             }
```

Comment:
Probably should throw here if it is not found? since sessionID will be nil then

Reply:
I have removed whole sessionId code since we don't have to pass sessionId while initializing SessionManager since it's always nil in start.